packages/core/src/hooks/useWizard/__stories__/useWizard.stories.module.scss

```diff
@@ -0,0 +1,6 @@
+.stepper {
```

Comment:
I'd stick to the current terminology

Reply:
Plus why not Flex?

Reply:
@talkor Notice it is being used inside
```
        <Steps
          className={styles.stepper}
```

I cannot use Flex

Reply:
what do you mean about the terminology, which one

Reply:
Oh ok nevermind, my bad, ignore